indico/modules/users/forms.py

```diff
@@ -48,10 +48,14 @@ class UserPreferencesForm(IndicoForm):
     timezone = SelectField(_('Timezone'))
     force_timezone = BooleanField(_('Use my timezone'),
                                   widget=SwitchWidget(),
-                                  description='Always use my current timezone instead of an event\'s timezone.')
+                                  description=_('Always use my current timezone instead of an event\'s timezone.'))
     show_past_events = BooleanField(_('Show past events'),
                                     widget=SwitchWidget(),
-                                    description='Show past events by default.')
+                                    description=_('Show past events by default.'))
+    use_previewer_pdf = BooleanField(
+        _('Use previewer for PDF files'),
```

Comment:
There's a clear lack of space for the whole line. Are you suggesting I change it in all of them?